rules/no-skip-test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+var util = require('../util');
+var createAvaRule = require('../create-ava-rule');
+
+module.exports = function (context) {
+	var ava = createAvaRule();
+	return ava.merge({
+		MemberExpression: function (node) {
+			if (ava.isTestFile && node.property.name === 'skip' && util.isCallFromTestObject(node)) {
```

Comment:
Ha! I thought there would be some kind of race condition if I put that in a CallExpression, but it's well written :D (though I think it'll break for nested tests. Don't know if that is currently possible with Ava now)
I'll add that anyway some time today!